pkgs/development/libraries/babl/default.nix

```diff
@@ -41,6 +41,10 @@ stdenv.mkDerivation (finalAttrs: {
 
   mesonFlags = [
     "-Dprefix-dev=${placeholder "dev"}"
+  ] ++ lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
+    # Docs are opt-out in native but opt-in in cross builds.
```

Comment:
Why is that?

Reply:
That sentence is meant to describe the upstream defaults. I guess I should clarify that.

Reply:
Oh I see